jaas/src/main/java/com/griddynamics/jagger/jaas/config/ApplicationConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.griddynamics.jagger.jaas.config;
+
+import com.griddynamics.jagger.dbapi.DatabaseService;
+import com.griddynamics.jagger.engine.e1.services.DataService;
+import com.griddynamics.jagger.engine.e1.services.DefaultDataService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.*;
+
+@PropertySources({@PropertySource("classpath:/basic/default.environment.properties")})
+@ImportResource({"classpath:/common/storage.rdb.client.conf.xml", "classpath:/spring/dbapi.config.xml"})
+@Configuration
+public class ApplicationConfig {
```

Comment:
No. You missed the point. The argue is about can you use Boot Starter as Root Config as well or have you to use a separate class for this. You said some best practises claim you have to use a separate AppConfig file for this. Where is this best practise described?